ios/TwilioVoiceReactNative.m

```diff
@@ -426,6 +426,14 @@ - (void)sendEventWithName:(NSString *)eventName body:(id)body {
     }
 }
 
+RCT_EXPORT_METHOD(voice_setCallKitConfiguration:(NSDictionary *)configuration
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    [self initializeCallKitWithConfiguration:configuration];
+    resolve(@"");
```

Comment:
Does this resolve `nil` for `Promise<void>`? Sorry, not familiar with the syntax of Obj-C.